### DISCUSSION TITLE
Sort tag list alphabetically

### DISCUSSION THREAD

**Discussion by jhemmje** [filipecarneiro/hugo-bootstrap-theme#14] [Q&A] [answered]
Hello,

how can I sort the tag list on the posts list site (e.g. https://filipecarneiro.github.io/hugo-bootstrap-theme/post/) alphabetically?
I have already consulted the documentation (https://gohugo.io/templates/taxonomy/#sort-alphabetically) and tried out the examples, but can't get it to work, because I don't know much about the Hugo/Go syntax.

BTW: Thanks for the great Theme! :)

**Comment by filipecarneiro** [comment, marked as answer]
Hi Jan-Hendrik,

Thank you for your feedback!

You can order a list of contents alphabetically with the [ByTitle](https://gohugobrasil.netlify.app/templates/lists/#order-content) method. 

Actually, in the header of the list page, it makes sense to order the categories and tags alphabetically. There's no an way to control their weight, without creating metadata for each possible category and tag.

In the list of tags of each page, for example, it's easy to control the order so, the default works fine.

 I've published a new version of the theme with this change. You can see the code in [this commit](https://github.com/filipecarneiro/hugo-bootstrap-theme/commit/18812f95e3cee31f70c29f97fb966a48f71fa2de).
 
 Thank you!
